### DISCUSSION TITLE
Change default view of source sontrol changes

### DISCUSSION THREAD

**Discussion by carloschneider** [gitkraken/vscode-gitlens#2757] [Q&A] [answered]
Hello guys.

Is it possible change the default view of `Source Control` `Changes` in my `settings.json` file?

![image](https://github.com/gitkraken/vscode-gitlens/assets/577143/cb491e61-351a-4878-ab74-927bbfe8907d)

**Comment by jkelroy** [comment, marked as answer]
Hello @carloschneider, Changes in Source Control is provided by VS Code, not by GitLens. I am not aware of a way to move Changes within Source Control.
